### PR TITLE
[Mentorship] Add the Swift Mentorship Program interest survey.

### PR DESCRIPTION
**Pull request by hborla**
Add the Swift Mentorship Program interest surveys, which will open on June 1st at 8am PST through June 26th at 12am PST. This change embeds the survey at a dedicated Swift.org link: https://www.swift.org/mentorship-interest-survey/

### CODE REVIEW

**Review by shahmishal — APPROVED** [review]
LGTM!

It's not a big deal, however the dark mode will not be respected.

File: mentorship-interest-survey/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Swift Mentorship Program Interest Survey
+---
+
+<iframe src="https://essentials.applesurveys.com/jfe/form/SV_bKovI2JRyJG3394" height="800px" width=100% frameBorder="0"></iframe>
```
Comment (alexandersandberg) [inline mentorship-interest-survey/index.md:6]: Some missing quotes here are causing this to appear as plain text. This should fix it. 🙂 

```suggestion
<iframe src="https://essentials.applesurveys.com/jfe/form/SV_bKovI2JRyJG3394" height="800px" width="100%" frameBorder="0"></iframe>
```

File: mentorship/index.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ After the 12-week cohort ends on September 30th, the interest surveys will open
 
 ### Interest Surveys
 
-The 2022 mentee and mentor interest survey submissions will open on June 1st - check back here for the survey links on that date. In the meantime, start to think through the specific goals that you'd like to work toward through the Swift Mentorship Program.
+The 2022 mentee and mentor interest survey submissions are open June 1st - June 15th. If you are interested in participating as a mentee or mentor, please fill out the interest survey [here](/mentorship-interest-survey/).
```
Comment (alexandersandberg) [inline mentorship/index.md:30]: Note: As you mentioned "June 1st at 8am PST through June 26th at 12am PST" in the PR description, it says "June 1st - June 15th" here.

File: mentorship-interest-survey/index.md (hunk 1 above)
Comment (hborla) [reply mentorship-interest-survey/index.md]: Thank you!

**Comment by hborla** [comment]
@swift-ci please test

**Review by alexandersandberg — APPROVED** [review]

**Review by 0xTim — APPROVED** [review]

**Pull request merged**